app/src/main/res/xml/video_audio_settings.xml

```diff
@@ -119,6 +119,13 @@
             app:singleLineTitle="false"
             app:iconSpaceReserved="false" />
 
+        <SwitchPreferenceCompat
```

Comment:
Do we really need to also add the setting here? Isn't the switch in the menu enough?

Reply:
Yes because it's not possible to reach that switch in the share dialog if a default value was chosen (e.g. always open with video player) :)